eventhubs/azure-spring-cloud-starter-eventhubs/eventhubs-integration/README.md

```diff
@@ -44,13 +44,15 @@ Running this sample will be charged by Azure. You can check the usage and bill a
    `application-mi.yaml` respectively.
     ```yaml
     spring:
-      cloud:
-        azure:
-          eventhub:
-            connection-string: [eventhub-namespace-connection-string]
-            checkpoint-storage-account: [checkpoint-storage-account]
-            checkpoint-access-key: [checkpoint-access-key]
-            checkpoint-container: [checkpoint-container]
+        cloud:
+          azure:
+              eventhubs:
+                  connection-string: [connection-string]
+                  processor:
+                      checkpoint-store:
```

Comment:
identation is wrong

Reply:
let's change the directory to new artifact id, and don't forget to modify the link path in readme.

Reply:
Remove this sentense: "Please note `Basic` tier is unsupported." in this file.

Reply:
updated